Resources/Prototypes/Entities/Structures/Doors/Firelocks/firelock.yml

```diff
@@ -126,3 +130,5 @@
           - MobImpassable
           - VaultImpassable
           - SmallImpassable
+    - type: PaintableAirlock
+      enabled: false
```

Comment:
Firelocks are different from airlocks, so they probably shouldn't be able to be repainted. Or their own colors if there are alternate firelock colorschemes.

Reply:
I created a bunch more groups, so now every airlock is grouped with other airlocks that share a similar shape.